python/src/equistore/operations/allclose.py

```diff
@@ -27,13 +29,13 @@ def allclose(
     ``allclose`` test with the provided ``rtol``, and ``atol``.
 
     In practice this function calls :py:func:`allclose_raise`, returning
-    ``True`` if no exception is rased, `False` otherwise.
+    ``True`` if no exception is raised, ``False`` otherwise.
```

Comment:
We should use be py:obj:`True` instead of ``True``. With this Sphinx will render this as a link to the Python documentation. We could also include [flake8-sphinx-links](https://pypi.org/project/flake8-sphinx-links) which will test this for us.

Reply:
Do we really need links for True/False? Other than that, sure I can add the links =)